packages/flutter/test/material/color_scheme_test.dart

```diff
@@ -45,4 +45,38 @@ void main() {
     expect(scheme.onError, const Color(0xff000000));
     expect(scheme.brightness, Brightness.dark);
   });
+
+  test('high contrast light scheme matches the spec', () {
+    const ColorScheme scheme = ColorScheme.highContrastLight();
```

Comment:
This would be a good place to reiterate the link to the spec that we're matching. Here and below.

Reply:
Done